### ISSUE TITLE
BUG: pypdpiper SGE executors die for long processing steps, leave pipeline stranded

### ISSUE THREAD

**Issue by gdevenyi** [Mouse-Imaging-Centre/pydpiper#153]
I'm running MBM on a SGE cluster.
My run submission was;
```
MBM.py --queue=sge --num-executors=240 --proc=1 --mem=4 --verbose --pipeline-name=rat --lsq6-large-rotations --registration-method=mincANTS --calc-stats --init-model=../MBM-input/model/A10FISP.mnc ../MBM-input/*mnc
```

The jobs executors seem to get paired down to the number of input files (25) and process away.

For some reason, one of the non-linear registrations too much longer than the others to complete, in that time, it looks like the other executors died off.

When the longer-running registration completed, no new executors were spawned, and the pipeline was left stranded with nothing to do the work.

**Comment by jasonlerch** [comment]
Is this issue still affecting anyone? Or has it been fixed in the development branch?

**Comment by gdevenyi** [comment]
I think that the fix for this bug introduced bug #172

**Comment by bcdarwin** [comment]
Closing for now; reopen if necessary.
